invenio_communities/communities/schema.py

```diff
@@ -174,6 +181,10 @@ class Meta:
         "is_verified": "moderate",
     }
 
+    field_load_permissions = {
+        "theme": "theme_create",
```

Comment:
specific permission to create theme. For dumping we don;t need because now there is a systemfield that pops the theme information before indexing.